src/ripple/consensus/Validations.h

```diff
@@ -192,6 +222,7 @@ class Validations
         decay_result_t<decltype (&Validation::ledgerID)(Validation)>;
     using NodeKey = decay_result_t<decltype (&Validation::key)(Validation)>;
     using NodeID = decay_result_t<decltype (&Validation::nodeID)(Validation)>;
```

Comment:
Yes, I should revisit this.  Two follow-ups

1. Is the `decay_result_t` style acceptable? Or do you prefer the [explicit `using`](https://github.com/bachase/rippled/blob/0be326e94a64110ce424a0fd7dced5b3903c20e4/src/ripple/app/consensus/RCLConsensus.h#L81-L84?) style?
2. If its the latter, I added the `_t` suffix because `NodeID` is an existing `ripple` type, so wasn't sure how to define an alias `using NodeID = NodeID`?